app/controllers/application_controller.rb

```diff
@@ -47,6 +47,14 @@ def routing_error
     # report the original 'cause' of the exception when present
     if SKIP_SENTRY_EXCEPTION_TYPES.include?(exception.class) == false
       extra = exception.respond_to?(:errors) ? { errors: exception.errors.map(&:to_hash) } : {}
+      if exception.is_a?(Common::Exceptions::BackendServiceException)
+        more_extra = { icn: current_user.try(:icn), mhv_correlation_id: current_user.try(:mhv_correlation_id) }
+        # Add additional user specific context to the logs
+        extra.merge!(more_extra) if current_user.present?
```

Comment:
Nitpicking but you could wrap both of these lines in the if test, remove the try calls and even remove the more_extra variable for a tiny cleanup